projects/Amlogic/packages/aml-s9xx-device-trees/package.mk

```diff
@@ -0,0 +1,49 @@
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2016 Mateusz Krzak (kszaquitto@gmail.com)
```

Comment:
Idk if you happy with that or not? 
Preferred is `Copyright (C) 2016-present Team LibreELEC` but I won't beat you to remove your copyright.

or maybe

```
Copyright (C) 2018-present Team LibreELEC
Copyright (C) 2016 Mateusz Krzak (kszaquitto@gmail.com)
```
whatever you like :)